WordPress/playground-tools
# Update Playground dependencies to 0.6.10

## What?



Update Playground dependencies

## Why?



We should keep all the projects updated, so it's easier to catch possible bugs and also give feedback to Playground core.
In Playground `php-wasm/node` `v0.6.9` we introduce the [throw JS errors when `php.run`](https://github.com/WordPress/wordpress-playground/pull/1137), but from my inspection it seems it won't affect `wp-now` or the `VS Code extension`

## How?



Updated package.json

## Testing Instructions






* Tests should pass
* Run `nvm use && npm install`
* Run `npx nx build wp-now && node dist/packages/wp-now/with-node-version.js start`
* Observe wp-now starts as expected
* Check other projects
